_includes/neurips/2019/presenters.nips.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
```

Comment:
In order for this _includes file to be displayed, it has to be tied to a page. I do not see a page created for presenters in this pull request. There will also need to be a link added to the navigation for this workshop.

See the readme for this site regarding file structure: https://github.com/latinxinai/latinxinai.github.io